From 1f549ee2df550c74209ca44b1dcb4db309d8c97f Mon Sep 17 00:00:00 2001
From: John Brooks <john.brooks@jollamobile.com>
Date: Mon, 17 Nov 2014 16:18:00 -0700
Subject: [PATCH] Remove serial-tests flag to fix automake 1.11

serial-tests was added in automake 1.12, but has no effect beyond the
default behavior until automake 1.13. Since we're still on 1.11, it
serves no purpose.

This will need to be reverted when we update to automake >= 1.13.
---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8a1939f..b6592ff 100644
--- a/configure.ac
+++ b/configure.ac
@@ -22,7 +22,8 @@ P11KIT_AGE=1
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIR([build/m4])
 AC_CONFIG_AUX_DIR([build/litter])
-AM_INIT_AUTOMAKE([1.12 foreign serial-tests subdir-objects])
+# automake versions 1.12 and later should have 'serial-tests' here, but it's incompatible with 1.11
+AM_INIT_AUTOMAKE([1.11 foreign subdir-objects])
 AM_SANITY_CHECK
 AM_MAINTAINER_MODE([enable])
 m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])],)
-- 
1.9.3

